neurogym/wrappers/components/parameters/activation.py

```diff
@@ -0,0 +1,108 @@
+import numpy as np
+import torch
+from torch import nn
+
+from neurogym.wrappers.components.layers.base import LayerMonitorBase
+
+
+class ActivationMonitor:
+    def __init__(self, module: nn.Module, steps: int, name: str, populations: set[str] | None = None):
```

Comment:
`steps` and `name` get defaulted in `record_activations`.
Do you think there's a chance this class is used from outside of that function? If so, maybe it would be good to set the defaults here instead.

Reply:
> Do you think there's a chance this class is used from outside of that function?

Hmm...unlikely but not impossible - I'll set the defaults here.👍